### PR TITLE
COMMERCE-10680 Fix "Two separate minicart items are created instead of a single one using quick add to cart along with the normal add to cart"

### PR DESCRIPTION
**Pull request by jaydawu**
https://issues.liferay.com/browse/COMMERCE-10680

Resend from https://github.com/liferay-commerce/liferay-portal/pull/3533

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCart.testcase [hunk 1]
```diff
@@ -170,6 +170,61 @@ definition {
...
+
+			CommerceNavigator.gotoMiniCart();
+
+			AssertTextEquals(
+				key_cartItemSku = "MIN93016A",
+				locator1 = "CommerceAccelerators#MINI_CART_ITEM_QUANTITY",
+				value1 = 1);
+		}
+
+		task ("And add the same SKU from the quick add-to-cart search bar with quantity 1") {
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCart.testcase:210]: And adding the same SKU ... 

It is better to keep the same verb tense

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCart.testcase [hunk 2]
```diff
@@ -170,6 +170,61 @@ definition {
...
+			FrontStore.quickAddToCartSearchAndSelectItem(orderItemSku = "MIN93016A");
+
+			AssertElementPresent(
+				locator1 = "CommerceWidget#TAG_FACET_LABEL_LAYOUT",
+				tagName = "MIN93016A");
+
+			Click(locator1 = "CommerceAccelerators#QUICK_ADD_TO_CART_BUTTON");
+		}
+
+		task ("Then assert the SKU in the cart is updated to quantity 2") {
```
Comment (22LucaGiannotta22) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCart.testcase:220]: Then the SKU in the cart is updated ...

**Comment by jaydawu** [comment]
ci:stop

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by jaydawu** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:forward

**Pull request closed without merging**